packages/core/src/components/Swiper/Swiper.tsx

```diff
@@ -89,6 +102,20 @@ export const SEED_DATA = {
       label: "Next Title",
       defaultValue: "",
     }),
+    prevTitleColor: createColorProp({
+      label: "Previous Title Color",
+    }),
+    nextTitleColor: createColorProp({
+      label: "Next Title Color",
+    }),
+    dotColor: createColorProp({
+      label: "Dot Color",
+      defaultValue: "#86939e",
```

Comment:
I don't think hex default values will work. It needs to be one of the theme colors e.g. `primary`, `secondary`, `error` etc. But run the script against staging, maybe I'm wrong